cli/lib/kontena/cli/stacks/service_generator.rb

```diff
@@ -27,7 +27,7 @@ def parse_data(options)
       data = {}
       data['instances'] = options['instances']
       data['image'] = parse_image(options['image'])
-      data['env'] = options['environment'] if options['environment']
+      data['environment'] = options['environment']
```

Comment:
Stack services are missing all of their envs.... the server param is still `data['env']`

No tests for this?

Reply:
There is now